content/docs/introduction/central.md

```diff
@@ -0,0 +1,68 @@
++++
+title = "Open mSupply Central Server"
+description = "Open mSupply central server functionality"
+date = 2022-06-10T11:38:00+00:00
+updated = 2022-06-10T11:38:00+00:00
+draft = false
+weight = 8
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+toc = true
+top = true
++++
+
+Open mSupply central server is a special site that allows for configurations of subsets of central data. Please see [requirements](/docs/introduction/requirements/#open-msupply-requirements) section for extra info.
```

Comment:
I didn't know what I was being redirected to - what I should look for on the requirements page.
How about:?

```suggestion
Open mSupply central server is a special site that allows for configuration of subsets of central data. Please see the [requirements](/docs/introduction/requirements/#open-msupply-requirements) section for details on requirements for running open mSupply, and the [Open mSupply central server](docs/introduction/requirements/#open-msupply-central-server) section for details about the central server requirements specifically.
```

Reply:
Open mSupply